src/_path.h

```diff
@@ -752,6 +752,10 @@ int count_bboxes_overlapping_bbox(agg::rect_d &a, BBoxArray &bboxes)
     agg::rect_d b;
     int count = 0;
 
+    // If bboxes is empty return zero
+    if (bboxes.dim(0) == 0 || bboxes.dim(1) == 0 || bboxes.dim(2) == 0)
```

Comment:
Maybe say: `bboxes.dim(1) != 2 || bboxes.dim(2) != 2`?  We are bound to have other issues if its mishapen in some other way.  Though having a zero anywhere in the shape doesn't indicate a bug elsewhere, having anything other than 2 in the final two dimensions *does*, so maybe we need to turn that case into an exception.